tests/test_measurements.py

```diff
@@ -1208,6 +1208,24 @@ def func():
         assert len(obs) == 1
         assert obs[0].return_type is State
 
+    @pytest.mark.torch
+    @pytest.mark.parametrize("dev_name", ["default.qubit", "default.mixed"])
+    def test_correct_density_matrix_torch(self, dev_name):
+        """Test that the correct density matrix is returned using torch interface."""
```

Comment:
Seems strange to have a single standalone test for PyTorch here. I think maybe `tests/devices/default_qubit_torch.py` would be better.

Also, do we test density matrices for the other interfaces already?

Reply:
I added more tests here (tf, jax), and it will be the case in the upcoming PRs 👍